R/query-string.R

```diff
@@ -8,29 +8,36 @@ queryStringFilter <- function(req){
 #' @importFrom utils URLdecode
 #' @noRd
 parseQS <- function(qs){
-  if (is.null(qs) || length(qs) == 0 || qs == ""){
+  # Is there data in the request?
```

Comment:
Now that this function supports arbitrary data -- not just query strings, what would you think about renaming the function `parseBody`? Or even leaving the old parseQS, introducing a new function for parsing JSON, and then having a higher-level function that could delegate to the appropriate parser as needed?